sidecar/config/flags.go

```diff
@@ -211,6 +212,11 @@ var Flags = []cli.Flag{
 		EnvVar: envVar(logLevelFlag),
 		Usage:  "Logging level (debug, info, warn, error, fatal, panic)",
 	},
+	cli.IntFlag{
```

Comment:
I think we wanted it left here so we explicitly show that port 6500 is used for the discovery api in order to avoid port conflicts